src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -759,4 +759,13 @@
   <data name="FileExplorerPreview_PreviewPane_GroupSettings.Text" xml:space="preserve">
     <value>Preview Pane</value>
   </data>
+  <data name="FileExplorerPreview_RunAsAdminRequired.Text" xml:space="preserve">
+    <value>You need to run as administrator to modify these settings.</value>

```

Comment:
To be in line with the text/hints on general settings page:

```suggestion
    <value>You need to run as administrator to modify these settings</value>
```

Reply:
@crutkas @niels9001 what is the guidance around using periods? I've seen some places in Windows Settings where it ends with a period and some where it doesn't.

Reply:
I think we should have a consequent user experience.